spree_multi_domain.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.require_path = 'lib'
   s.requirements << 'none'
 
-  version = '~> 3.0.0'
+  version = '~> 3.1.0.beta'
```

Comment:
Incorrect, this is the `master` branch you should submit your PR to `3-0-stable` branch instead.